warehouse/templates/manage/project/settings.html

```diff
@@ -102,7 +102,7 @@ <h3>{% trans %}<abbr title="two factor authentication">2FA</abbr> requirement{%
     <h3>{% trans %}Project description and sidebar{% endtrans %}</h3>
     <p>
       {% trans project_name=project.name, setup_args_href='https://packaging.python.org/tutorials/distributing-packages/#setup-args', twine_docs_href='https://twine.readthedocs.io/en/latest/', distribution_href='https://packaging.python.org/tutorials/distributing-packages/' %}
```

Comment:
On this line `setup_args_href` refers to `setup.py` arguments, which is then used in the content line. 
Rename to something like `pyproject_args_href` and update the target URL to a correct anchor in the documentation, and then use the new name in line 105 `<a href=` tag

Reply:
#13310 Hello please check this. Done. If anything else happy to work it out.